matrix_3x3_3.0.cpp

```diff
@@ -0,0 +1,343 @@
+﻿#include <stdio.h>
+#include <stdlib.h>
+#include<time.h>
+
+typedef struct
+{
+	float data[3][3];
+	int width;
+	int height;
+} Matrix;
+
+void print_matrix(Matrix* M)
+{
+	for (int row_num = 0; row_num < M->height; row_num++)
+	{
+		for (int col_num = 0; col_num < M->width; col_num++)
+		{
+			printf("%f\t", M->data[row_num][col_num]);
+		}
+		printf("\n");
+	}
+	printf("\n");
+}
+
+Matrix READ_mat(char* name, int width, int height)

```

Comment:
Could use more speaking names : 
READ_mat --> readFileToMatrix
name --> filePath
width --> xSize / matrixXSize / matrixWidth / ?
height --> ySize / matrixYSize / matrixHeight / ?

It is in general not very good, if you try to do several things at once : you read text from file and construct Matrix out of it. This makes your readFile() function not reusable.